metricbeat/module/doc.go

```diff
@@ -34,7 +34,8 @@ List of standardised words and units across all metricsets. On the left are the
 * pct: precentage
 * request: req
 * sec: seconds, second, s
-*
+* ms: millisecond, millis
+* mb: megabytes
```

Comment:
one more we could add here that I spotted in your code and like is `msg` for `message`.